frontend/vite-project/src/pages/ProductDetails.tsx

```diff
@@ -84,18 +99,17 @@ const ProductDetails: React.FC = () => {
               Tasks for {plant.product_name}
             </h4>
             <ul className="product-tasks__items">
-              <li>
-                <strong>Task:</strong> This is some placeholder text to test
-                styling.
-              </li>
-              <li>
-                <strong>Another Task:</strong> This is some more placeholder
-                text.
-              </li>
-              <li>
-                <strong>One More Task:</strong> This is one last placeholder
-                text to test styling.
-              </li>
+              {/* --- Dynamic rendering of tasks --- */}
+              {tasks.length > 0 ? (
+                tasks.map((task) => (
+                  <li key={task.id} className="product-task-item">
+                    <span role="img" aria-label="task">🌱</span>
+                    <strong>Task:</strong> {task.description}
```

Comment:
```suggestion
                    <strong>{months[task.start_month]} to {months[task.end_month]}</strong> {task.description}
                  </li>
```

You'll need to add some additional logic here in cases where the task starts and ends on the same month. I would have it default to the zero index which is a blank string, but again, there may be better ways to do it.